deploy/servctl_utils/yaml_settings_utils.py

```diff
@@ -0,0 +1,107 @@
+import collections
+import io
```

Comment:
This file is copy-pasted from the piepeline repo, so theres no need to code review it: https://github.com/broadinstitute/hail-elasticsearch-pipelines/blob/master/kubernetes/yaml_settings_utils.py